lib/HTFeed/Storage/ObjectStore.pm

```diff
@@ -201,5 +212,10 @@ sub record_backup {
   return $rval;
 }
 
+sub audit_path {
+  my $self = shift;
+
+  return "s3://$self->{s3}{bucket}/" . $self->object_path;
```

Comment:
Based on the need in HT-3057 we may end up just putting this in `object_path`, but I think this is correct for right now.